backend/src/main/java/com/woowacourse/naepyeon/config/logging/LoggingInterceptor.java

```diff
@@ -20,29 +19,18 @@ public class LoggingInterceptor implements HandlerInterceptor {
     private final ObjectMapper objectMapper;
 
     @Override
-    public void afterCompletion(
-            final HttpServletRequest request,
-            final HttpServletResponse response,
-            final Object handler,
-            final Exception ex
-    ) throws Exception {
-        final ContentCachingRequestWrapper cachingRequest = (ContentCachingRequestWrapper) request;
-
-        if (isFailed(response.getStatus())) {
-            return;
-        }
+    public boolean preHandle(final HttpServletRequest request, final HttpServletResponse response, final Object handler)
+            throws Exception {
+        final CustomCachingRequestWrapper cachingRequest = (CustomCachingRequestWrapper) request;
```

Comment:
```suggestion
        final CustomCachingRequestWrapper cachingRequest = WebUtils.getNativeRequest(request, ContentCachingRequestWrapper.class); 
        
        if (cachingRequest == null) {
            log.info("json형태의 데이터가 아닙니다");
            return;
        }
```

해당 CustomCachingRequestWrapper로 캐스팅하는 부분은 requset가 json데이터인걸 가정하고 하는 것 같습니다. 만약에 json데이터가 아니라면 ClassCastException이 발생할 것 같습니다. ex) MultipartFile이 담긴 form데이터 StandardMultipartHttpServletRequest나, restDocs에서 사용하고 있는 MockHttpServletRequest같은 데이터는 캐스팅 할 수 없습니다.

따라서 WebUtils를 활용해서 캐스팅할 수 있으면 해당 객체로 캐스팅하고 없다면 null을 반환시켜서 어떠한 후 작업을 처리해주는건 어떨까요?

Reply:
문제 확인하고 코드 변경해봤는데 다시 한번 확인 부탁드려요!